ddtrace/tracer/span.go

```diff
@@ -441,11 +446,15 @@ func (s *Span) setTagError(value interface{}, cfg errorConfig) {
 			s.setMeta(ext.ErrorDetails, fmt.Sprintf("%+v", v))
 			if !cfg.noDebugStack {
 				s.setMeta(ext.ErrorStack, err.Format())
+				s.setMeta(ext.ErrorHandlingStack, takeStacktrace(cfg.stackFrames, cfg.stackSkip))
 			}
 			return
 		}
 		if !cfg.noDebugStack {
-			s.setMeta(ext.ErrorStack, takeStacktrace(cfg.stackFrames, cfg.stackSkip))
+			telemetry.Count(telemetry.NamespaceTracers, "errorstack.source", []string{"source:takeStacktrace"}).Submit(1)
```

Comment:
`takeStacktrace` is now being called from line 449 to calculate the handling stack. In order to avoid double counting how many errors are relying on `takeStacktrace` to get the error stack, we move the telemetry counter from line 466 to here.